python/pyspark/sql/dataframe.py

```diff
@@ -1364,7 +1364,9 @@ def subtract(self, other):
         """ Return a new :class:`DataFrame` containing rows in this frame
         but not in another frame.
 
-        This is equivalent to `EXCEPT` in SQL.
+        This is equivalent to `EXCEPT DISTINCT` in SQL.
+
+        (Note: Before Spark 2.0, the behavior was equivalent to `EXCEPT ALL` in SQL.)
```

Comment:
In PySpark, we can use `.. note:: `. This makes the doc pretty :).

Reply:
nit: `2.0` to `2.0.0`

Reply:
Actually, before 2.0, it is not equivalent to EXCEPT ALL. For details, see the PR: https://github.com/apache/spark/pull/12736